evento2018.html

```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
 
   <div id="objetivo" class="row ">
     <div class="col-md-12  text-center ">
-      <h1>Nuestro Objetivo...</h1>
+      <h1>Our objective...</h1>
       <p>
         <span class="rojo bold">></span>
-        Somos una Comunidad que cree firmemente que en el interior de cada persona existe un potencial inigualable para crear y afectar
-        positivamente a las personas y al entorno que nos rodea, más aún cuando esa capacidad de crear se suma a la de otros,
-        comprendiendo asi que el todo es más que
-        <b>la suma de las partes</b>
+        We are a Community that firmly believes that within each person there is an unparalleled potential to create and affect positively to the people and the environment that surrounds us, especially when that capacity to create adds to that of others, understanding that the whole is more than <b> the sum of the parts </b>
```

Comment:
`affect positively the people and the environment that surrounds us`
`sum of its parts`